mysql-test/main/secondary_key_costs.test

```diff
@@ -1,6 +1,7 @@
 --source include/have_sequence.inc
 --source include/not_embedded.inc
 --source include/have_innodb.inc
+--source include/not_valgrind.inc
```

Comment:
Hereby we are losing Valgrind memcheck coverage of some earlier tests in this file. It would be useful to have a comment here that explains why we are doing it. Is it only because Valgrind is extremely slow compared to `-fsanitize=address` and `-fsanitize=memory` instrumented builds?

Reply:
Yes, valgrind is very slow on this test.  
Actually not due to this testcase, due to MDEV-33306.

Reply:
Added a note about this